readthedocs/proxito/views/serve.py

```diff
@@ -691,7 +689,7 @@ def _get_custom_404_page(self, request, project, version=None):
                     storage_root_path, tryfile
                 )
                 if build_media_storage.exists(storage_filename_path):
-                    log.info(
+                    log.debug(
                         "Serving custom 404.html page.",
```

Comment:
Do we have too many of these? I don't think too many people have custom 404 pages. This is actually useful to know what those projects are.

Not super relevant, but just nothing that I've used this log line in the past.

Reply:
@humitos It's our most common log message, as shown in the chart above.

Reply:
Wow 😳. OK.

The chart didn't load the first time I opened the PR 🤷‍♂️